### ISSUE TITLE
Consider removing the toast showing Min/Max time

### ISSUE THREAD

**Issue by fpaaske** [davecoffin/nativescript-modal-datetimepicker#75]
This is not really a bug, but more a suggestion/change request.

I'm currently using version **1.2.4** of this plugin in a **NativeScript 6** app.
When I'm opening a time picker dialog with max or min time set in `PickerOptions`, a toast is shown with this information

![image](https://user-images.githubusercontent.com/1579581/97881113-45ad2200-1d22-11eb-94cc-21c890f9b4e2.png)

Unfortunately, this text is not translatable and it looks weird to show this one message in English when the rest of the app is in a different language.

Please consider removing the toast.

Edit: I also see that 1.2.4 is not in any commit in this repo? Where is that release coming from?

**Comment by shiv19** [comment]
Good suggestion! We can add an option to turn off the toast and also to configure the text on it.

**Comment by fpaaske** [comment]
Yes, that sounds great! Hopefully it can be done on both version 1.x and 2.x :)

**Comment by fpaaske** [comment]
@shiv19 @davecoffin did you get time to look at this?

**Comment by syahnur197** [comment]
quick and dirty way is to comment out line 120-122 in `node_modules/nativescript-modal-datetimepicker/modal-datetimepicker.android.js` in your node_modules

```
// if (toastMsg !== "") {
//    android.widget.Toast.makeText(Application.android.foregroundActivity, toastMsg, android.widget.Toast.LENGTH_LONG).show();
// }
```
